introduction.md

```diff
@@ -0,0 +1 @@
+The introduction for this dojo. Not every sensei should be respected. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.